### PR TITLE
Add BSD 2-clause license

### PR DESCRIPTION
**Pull request by gmgunter**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2020, Caesar Developers
```
Comment (gmgunter) [inline LICENSE]: > Caesar Developers

What to put here?

Comment (r-burns) [reply LICENSE]: I like `Caesar authors & contributors`

**Comment by aivazis** [comment]
might as well start with a year range, e.g. 2020-2021, since the new year is right around the corner.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2020-2021, Caesar authors & contributors
```
Comment (r-burns) [inline LICENSE]:
```suggestion
Copyright 2020-2021, Caesar authors & contributors
```
the (c) is meaningless, just copyright is sufficient

Comment (gmgunter) [reply LICENSE]: Just copy-pasted the github-supplied template :shrug:

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright 2020-2021, Caesar authors & contributors
+All rights reserved.
```
Comment (r-burns) [inline LICENSE]:
```suggestion
```
same goes for this phrase

Comment (gmgunter) [reply LICENSE]: I'm happy to remove it but the [other](https://github.com/Homebrew/brew/blob/master/LICENSE.txt) [BSD 2-clause](https://github.com/ponylang/ponyc/blob/master/LICENSE) [repos](https://github.com/go-redis/redis/blob/master/LICENSE) that I see all include this verbiage from the template script. 

Is there any particular reason to remove this language other than it being basically meaningless? Leaving it unmodified seems simpler and more fool-proof to me as a non-expert.

Comment (r-burns) [reply LICENSE]: hmm, I don't see that language on wikipedia or opensource.org
https://en.wikipedia.org/wiki/BSD_licenses#2-clause_license_("Simplified_BSD_License"_or_"FreeBSD_License")
https://opensource.org/licenses/BSD-2-Clause
weird to me that most repos have it but those without also work with the license detector
https://github.com/cyfdecyf/cow
https://github.com/insidegui/WWDC

Comment (gmgunter) [reply LICENSE]: OK it's clearly not necessary. I'll remove it. :+1:

**Comment by gmgunter** [comment]
When you guys get a chance, can you take another look and let me know if this is OK to merge. I was waiting to merge the license before pushing any actual code.

**Review by aivazis — APPROVED** [review]

**Review by r-burns — APPROVED** [review]

**Pull request merged**